src/ui/settings/colorMode.js

```diff
@@ -4,6 +4,8 @@
 import { jsx, useColorMode } from 'theme-ui';
 import { Button } from '@theme-ui/components';
 import { useTranslation } from 'react-i18next';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faMoon, faSun } from "@fortawesome/free-solid-svg-icons";
```

Comment:
Not sure if that's consistent in the existing code, but the second import should probably also use single quotes.